MODULE.bazel

```diff
@@ -24,6 +24,7 @@ use_repo(node, "nodejs_linux_arm64")
 use_repo(node, "nodejs_linux_ppc64le")
 use_repo(node, "nodejs_linux_s390x")
 use_repo(node, "nodejs_windows_amd64")
+use_repo(node, "nodejs_windows_arm64")
```

Comment:
This requires https://github.com/bazel-contrib/rules_nodejs/pull/3846 to be merged first.